backend/demo/src/main/java/com/example/demo/services/ProductService.java

```diff
@@ -0,0 +1,87 @@
+package com.example.demo.services;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.List;
+import java.util.Optional ; 
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.extern.slf4j.Slf4j;
+
+import com.example.demo.model.Product;
+import com.example.demo.repository.ProductRepository;
+@Slf4j
+@Service
+public class ProductService {
+
+	@Autowired
+	private ProductRepository productRepository;
+	
+	
+	public List<Product> getallProduct () {
```

Comment:
always use Camel Case when naming methods --> getAllProducts is more suitable, also notice the s at the end of methods returning lists it is more readable